packages/preview-middleware-client/src/adp/quick-actions/fe-v2/lr-enable-semantic-date-range-filter-bar.ts

```diff
@@ -33,8 +33,19 @@ export class ToggleSemanticDateRangeFilterBar
             const isActionApplicable = pageHasControlId(this.context.view, control.controlId);
             const modifiedControl = getControlById<FilterBar>(control.controlId);
             if (isActionApplicable && modifiedControl) {
-                this.isUseDateRangeTypeEnabled = modifiedControl.getProperty('useDateRangeType');
                 this.control = modifiedControl;
+
+                // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+                const id: string = this.control.getProperty('persistencyKey') ?? this.control.getId();
```

Comment:
If you typecast this via `as string` you can get rid of the eslint disablement comment